doc/ReleaseNotes

```diff
@@ -31,6 +31,16 @@ TCPDirect distributed as source code
   packages are suitable for all supported OSs.
 
 
+New onload-devel package required
+---------------------------------
+
+  Onload now includes a package containing all of the header files used by
+  Onload. Installing this package is now required to build TCPDirect applications.
+
+  Headers can also be installed via new '--headers' option in 'onload_install'
```

Comment:
Maybe omit this? TCPDirect-9 will be provided as SRPM/ source DEB only. So my current expectation is that we want the customer to use the same type of Onlaod-9 package?

Reply:
It can be downloaded from the Git checkout no? And a tarball generated from the checkout?

Reply:
And its safe to assume customers will not always use the same package type